runtime/interpreter/encode.go

```diff
@@ -1645,4 +1681,16 @@ func (e EmptyTypeInfo) Encode(encoder *cbor.StreamEncoder) error {
 	return encoder.EncodeNil()
 }
 
+func (e EmptyTypeInfo) IsComposite() bool {
+	return false
+}
+
+func (e EmptyTypeInfo) Identifier() string {
+	return "null"
```

Comment:
Can this maybe just be an empty string `""` to avoid the overhead of storing the literal `"null`"?